pkg/resources/prometheus/configmap-rules.go

```diff
@@ -274,8 +274,8 @@ groups:
     labels:
       kubermatic: federate
 
-  - record: job:machine_controller_machines:sum
-    expr: sum(machine_controller_machines)
+  - record: job:machine_controller_machines_total:rate5m
+    expr: rate(machine_controller_machines_total[5m])
```

Comment:
Why did you change this?

Reply:
The metric name was changed from `machine_controller_machines` to `machine_controller_machines_total`

Reply:
https://github.com/kubermatic/machine-controller/blob/master/pkg/controller/machine/metrics.go#L104